expression/builtin_string_test.go

```diff
@@ -1429,15 +1430,15 @@ func TestBitLength(t *testing.T) {
 }
 
 func TestChar(t *testing.T) {
-	t.Parallel()
```

Comment:
We have `expression/builtin_string_serial_test.go`, so not sure where this test case should be after delete `t.Parallel()` :(